packages/amplify-graphql-model-transformer/src/resources/custom-resource-lambda/custom-resource-handler.ts

```diff
@@ -0,0 +1,440 @@
+import { DynamoDB } from 'aws-sdk';
```

Comment:
nit on overall naming. I see you using `custom-resource`, `managed-table`, and `iterative-deployment`. Let's coalesce around a single name scheme for class names, resource names, and filenames (To make it easier to understand how this all works together).

I think I might prefer `DynamicTable` or something like that, to indicate it's not static within the context of the CDK application definition. @palpatim if you have any suggestions too.

Reply:
Discussed offline, and at least temporarily going w/ `AmplifyTable`. here.